lib/_stream_readable.js

```diff
@@ -729,33 +729,37 @@ Readable.prototype.pipe = function(dest, pipeOpts) {
 
   src.on('data', ondata);
   function ondata(chunk) {
-    debug('ondata');
-    const ret = dest.write(chunk);
-    debug('dest.write', ret);
-    if (ret === false) {
-      // If the user unpiped during `dest.write()`, it is possible
-      // to get stuck in a permanently paused state if that write
-      // also returned false.
-      // => Check whether `dest` is still a piping destination.
-      if (!cleanedUp) {
-        if (state.pipes.length === 1 && state.pipes[0] === dest) {
-          debug('false write response, pause', 0);
-          state.awaitDrainWriters = dest;
-          state.multiAwaitDrain = false;
-        } else if (state.pipes.length > 1 && state.pipes.includes(dest)) {
-          debug('false write response, pause', state.awaitDrainWriters.size);
-          state.awaitDrainWriters.add(dest);
+    try {
+      debug('ondata');
+      const ret = dest.write(chunk);
+      debug('dest.write', ret);
+      if (ret === false) {
```

Comment:
```suggestion
      if (!ret) {
```

Reply:
This change is not relevant for this PR. It's just whitespace changes.